pom.xml

```diff
@@ -240,6 +240,11 @@
     <spark.test.home>${session.executionRootDirectory}</spark.test.home>
 
     <CodeCacheSize>1g</CodeCacheSize>
+
+    <!-- Thrift properties -->
+    <thrift.home>you-must-set-this-to-run-thrift</thrift.home>
```

Comment:
Does it need to be set, or can Maven find thrift from PATH? Or even better, do the generation via libthrift from maven central itself?